### ISSUE TITLE
Incorrect error: `<th aria-sort="XXX">`

### ISSUE THREAD

**Issue by SaekiTominaga** [markuplint/markuplint#397]
The following is valid HTML, but markuplint gives a `wai-aria` error.

```html
<table>
  <thead>
    <tr>
      <th aria-sort="ascending"></th>
    </tr>
  </thead>
</table>
```

`aria-sort` spec:
- [WAI-ARIA 1.1](https://www.w3.org/TR/wai-aria-1.1/#aria-sort)
- [WAI-ARIA 1.2](https://www.w3.org/TR/wai-aria-1.2/#aria-sort)

**Comment by YusukeHirao** [comment]
@SaekiTominaga 

Thank you. 
Fixed on [v2.2.0](https://github.com/markuplint/markuplint/releases/tag/v2.2.0)

**Comment by YusukeHirao** [comment]
`aria-sort` is:

- Use in Roles:
  - [columnheader](https://www.w3.org/TR/wai-aria-1.1/#columnheader)
  - [rowheader](https://www.w3.org/TR/wai-aria-1.1/#rowheader)

- [columnheader](https://www.w3.org/TR/wai-aria-1.1/#columnheader) Base Concept: [HTML th[scope="col"]](https://html.spec.whatwg.org/multipage/tables.html#attr-th-scope)
- [rowheader](https://www.w3.org/TR/wai-aria-1.1/#rowheader) Base Concept: [HTML th[scope="row"]](https://html.spec.whatwg.org/multipage/tables.html#attr-th-scope)

The role of the th element without the scope attribute is `cell`.

**Comment by YusukeHirao** [comment]
> The role of the th element without the scope attribute is `cell`.

It was incorrect.
The `th` element is the `columnheader` role if it descendants the `thead` element.
The `th` element is the `rowheader` role if it descendants the `tbody` element.
